ops/prod-deploy.tmpl.yaml

```diff
@@ -51,11 +51,11 @@ extraEnvVars: &envVars
   - name: FCREPO_BASE_PATH
     value: /prod
   - name: FCREPO_HOST
-    value: 52.90.209.247:8080
+    value: 10.0.4.93:8080
```

Comment:
☝️ why are we switching from public to private here?

Reply:
they are in the same aws availability zone / account now so internal traffic is a) free and b) faster.

Reply:
👍